projects/packages/search/src/dashboard/components/record-meter/index.jsx

```diff
@@ -24,6 +27,12 @@ import './style.scss';
 export default function RecordMeter() {
 	const tierMaximumRecords = useSelect( select => select( STORE_ID ).getTierMaximumRecords() );
 
+	// this.state = {
+	// 	recordInfo: getRecordInfo(createData().data, createData().planInfo),
+	//   };
+
+	const [ state ] = useState( getRecordInfo( createData().data, createData().planInfo ) );
```

Comment:
I _think_ what we need here is something like:

```
const [ recordInfo, setRecordInfo ] = useState( getRecordInfo( createData().data, createData().planInfo ) );
```

...and then you should be access the state using the `recordInfo` variable.

Reply:
Yep, this works nicely!